apps/designer/app/designer/features/style-panel/controls/menu/menu-control.tsx

```diff
@@ -41,13 +41,20 @@ export const MenuControl = ({
       };
     })
     .filter((item) => item.icon);
+  let variant: "default" | "set" | "inherited" = "default";
+  if (styleSource === "local") {
+    variant = "set";
+  }
+  if (styleSource === "remote") {
+    variant = "inherited";
+  }
```

Comment:
We need to unify this logic and make all controls take a `color` prop

Reply:
we should also start using semantic colors for these (ds in figma has fully switched to semantic colors, we haven't migrated yet but we can start one by one)

Reply:
Color is too optimistic to call it. These are still variants with multiple colors. And there would be 2 blue for "set/local" and "active". We just need to switch to new terms.

Reply:
Anyway let's do this in other PR

Reply:
color prop is meant to be interpreted as a "color scheme", we researched all options there